src/functions/template-tags/organizer.php

```diff
@@ -265,11 +265,28 @@ function tribe_get_organizer_link( $postId = null, $full_link = true, $echo = fa
 
 		$org_id = tribe_get_organizer_id( $postId );
 		if ( class_exists( 'Tribe__Events__Pro__Main' ) && get_post_status( $org_id ) == 'publish' ) {
-			$url = esc_url_raw( get_permalink( $org_id ) );
+			$url    = esc_url_raw( get_permalink( $org_id ) );
+			$target = apply_filters( 'tribe_get_event_organizer_link_target', '_self' );
+			$rel    = ( '_blank' === $target ) ? 'noopener noreferrer' : '';
+
 			if ( $full_link ) {
 				$name = tribe_get_organizer( $org_id );
-				$attr_title = the_title_attribute( array( 'post' => $org_id, 'echo' => false ) );
-				$link = ! empty( $url ) && ! empty( $name ) ? '<a href="' . esc_url( $url ) . '" title="'.$attr_title.'">' . $name . '</a>' : false;
+
+				if ( empty( $url ) || empty( $name ) ) {
+					$link = false;
+				} else {
+					$link = sprintf(
+						'<a href="%s" title="%s" target="%s" rel="%s">%s</a>',
+						esc_url( $url ),
+						the_title_attribute( [
+							'post' => $org_id,
+							'echo' => false
+						] ),
```

Comment:
:no_entry_sign: Closing parenthesis of a multi-line function call must be on a line by itself

<sub>hash: 792c23f6abb49617d728f5cfb9001554</sub>